internal/configuration/settings/storage.go

```diff
@@ -0,0 +1,50 @@
+package settings
+
+import (
+	"github.com/qdm12/gluetun/internal/constants"
+	"github.com/qdm12/gosettings"
+	"github.com/qdm12/gosettings/reader"
+	"github.com/qdm12/gotree"
+)
+
+// StorageSettings contains settings to configure the storage.
+type StorageSettings struct {
+	// Filepath is the path to the servers.json file. An empty string disables on-disk storage.
+	Filepath *string
+}
+
+func (s StorageSettings) validate() (err error) {
+	return nil
+}
+
+func (s *StorageSettings) copy() (copied StorageSettings) {
+	return StorageSettings{
+		Filepath: s.Filepath,
+	}
+}
+
+// overrideWith overrides fields of the receiver
+// settings object with any field set in the other
+// settings.
+func (s *StorageSettings) overrideWith(other StorageSettings) {
+	s.Filepath = gosettings.OverrideWithPointer(s.Filepath, other.Filepath)
+}
+
+func (s *StorageSettings) setDefaults() {
+	s.Filepath = gosettings.DefaultPointer(s.Filepath, constants.ServersData)
+}
+
+func (s StorageSettings) String() string {
+	return s.toLinesNode().String()
+}
+
+func (s StorageSettings) toLinesNode() (node *gotree.Node) {
+	node = gotree.New("Storage settings:")
+	node.Appendf("Filepath: %s", *s.Filepath)
+	return node
+}
+
+func (s *StorageSettings) read(r *reader.Reader) (err error) {
+	s.Filepath = r.Get("STORAGE_FILEPATH", reader.AcceptEmpty(true))
```

Comment:
Please add `STORAGE_FILEPATH=/gluetun/servers.json` to the Dockerfile in the `ENV` section, thanks!

Reply:
I left it out because when the setting is unset it will use the default. That's also why I had to use a pointer -- not set means use default, empty string means disable writing. In any case, I can certainly add the setting with its default value in the dockerfile.

Reply:
> I left it out because when the setting is unset it will use the default.

Indeed, but this is for documentation purposes for the Docker image, to show what's the default.

Reply:
My bad, I just re-read this, I previously misunderstood. I think `reader.AcceptEmpty(true)` does solve it indeed, good catch! My apologies for not pointing you in that direction!